.neuro/live.yaml

```diff
@@ -7,36 +7,36 @@ defaults:
 
 volumes:
   data:
-    remote: storage://neuro-compute/yevheniisemendiak/ml_recipe_bone_age/data
+    remote: storage://onprem-poc/alexeynayden/data
```

Comment:
please use `storage:ml_recipe_bone_age/data` -- it's generalized values (for neuro-compute) that we will introduce in https://github.com/neuro-inc/ml-recipe-bone-age/pull/18